blog/_posts/2016-05-27-scala-center-advisory-board.md

```diff
@@ -0,0 +1,117 @@
+meetings in the future.
+
+## How does the board work?
+
+The main goal for each meeting is to define a set of tasks - we call these
+"recommendations" - for the Scala Center to undertake over the coming quarter.
+Each board member can submit proposals for discussion - short overviews of a
+change or project they would like to see the Scala Center use its resources to
+implement - and a few minutes (usually no more than ten) will be devoted to
+discussing that topic amongst the members.
+
+An important aspect of the relationship between the Scala Center and the
+Advisory Board is that recommendations are not binding. This means that the
+Scala Center has the option to ignore recommendations, if it decides to, which
+may seem surprising, but is actually very important. It not only maintain
+independence between the two bodies, but also in ensuring that the
+recommendations received from the Advisory Board are within our budgetary and
+technical capabilities. And this is why Heather and Martin sit on the board, in
+a non-voting capacity: to offer advice to the members on the viability of each
+of the proposals under discussion, so that the debates can focus on making a
+coherent set of recommendations which is actually achievable.
+
+For example, a recommendation to halve scalac compilation times may have the
+support of the entire Advisory Board, but the Scala Center would struggle to
+make it happen, so there would be little point in the Advisory Board making it
+a recommendation.
+
+So the discussions we have around each proposal may more closely resemble a
+*negotiation* whereby the scope of a proposal is narrowed (or maybe widened) to
+garner the support necessary from the board, and confirmation from the
+Executive Director and Technical Advisor that the idea is viable.
+
+Once there's some consensus around the table, we move to vote on the proposal
+and have a show-of-hands to formally decide whether to adopt the proposal as a
+recommendation. For some proposals, we may not reach any agreement amongst the
+members in the time available, in which case, the chairperson may decide to
+defer further discussion until the next meeting, or suggest that the proposal
+is rewritten and resubmitted at the next meeting.
+
+I mentioned at the beginning that it's important for the Advisory Board to make
+good on its promise, but it's just as important that we are *seen* to be doing
+so.  So the whole Advisory Board process is open, and we will publish the
+proposals and minutes from each meeting. You will be able to see these in the
+Advisory Board's [Github
+repository](https://github.com/scalacenter/advisoryboard/) in the next few
+days.
+
+## The First Meeting
+
+During the first meeting, we also elected Seth Tisue as the secretary, and his
+minutes will be published in the next few days. The meeting itself was a
+success. We deferred one proposal until the next meeting (which will take place
+in August), but adopted four others.
```

Comment:
Not required, but a suggestion nonetheless: should we hint at the recommendations that were made? Just make a short bulleted list and defer to the minutes for the details?